root/release/edit/information.tt

```diff
@@ -27,7 +27,11 @@
         <td class="release-artist" data-bind="artistCreditEditor: $data"></td>
       </tr>
 
-      [% table_row_error(2, 'showErrorWhenTabIsSwitched: needsArtistCredit', l('You haven’t entered a complete artist credit.')) %]
+      [% table_row_error(
+        2,
+        'showErrorWhenTabIsSwitched: needsArtistCredit',
+        l('You must select an artist for the release (the artist background should be green). If you have entered an artist but the background is white, click the magnifying glass to match the name to a MusicBrainz artist. If the artist doesn’t exist yet, you can create it from the search results.'),
```

Comment:
I think that "the artist background should be green" could be slightly more specific ("background" on its own feels like design or CSS terminology to me). What do you think?

```suggestion
        l('You must select an artist for the release (the background color of the artist input field should be green). If you’ve entered an artist but the background is white, click the magnifying glass to match the name to a MusicBrainz artist. If the artist doesn’t exist yet, you can create it from the search results.'),
```

Reply:
Sure, made the change.